cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts

```diff
@@ -1,93 +1,268 @@
+      workloadsPodPage.sortableTable().pagination().checkNotExists();
+    });
+  });
+
+  describe('Should open a terminal', () => {
+    beforeEach(() => {
+      workloadsPodPage.goTo();
+    });
+
+    it('should open a pod shell', () => {
+      const shellPodPo = new PodPo();
+
+      shellPodPo.openPodShell();
+    });
+  });
+
+  describe('When cloning a pod', () => {
+    const { name: origPodName, namespace } = createPodBlueprint.metadata;
+    const { name: clonePodName } = clonePodBlueprint.metadata;
+
+    beforeEach(() => {
+      cy.intercept('GET', `/v1/pods/${ namespace }/${ origPodName }?exclude=metadata.managedFields`).as('origPod');
+      cy.intercept('GET', `/v1/pods/${ namespace }/${ clonePodName }?exclude=metadata.managedFields`).as('clonedPod');
+
+      workloadsPodPage.goTo();
+
+      const createPodPo = new PodPo();
+
+      createPodPo.createPodViaKubectl(createPodBlueprint);
+    });
+
+    it(`Should have same spec as the original pod`, () => {
+      const cloneCreatePodPage = new WorkLoadsPodDetailsPagePo(origPodName, { mode: 'clone' });
+
+      cloneCreatePodPage.goTo();
+
+      let origPodSpec: any;
+
+      cy.wait('@origPod', { timeout: 20000 })
+        .then(({ response }) => {
+          expect(response?.statusCode).to.eq(200);
+          origPodSpec = response?.body.spec;
+          expect(origPodSpec.containers[0].resources).to.deep.eq(createPodBlueprint.spec.containers[0].resources);
+        });
+
+      const createClonePo = new PodPo();
+
+      // Each pod need a unique name
+      createClonePo.nameNsDescription().name().set(clonePodName);
+      createClonePo.save();
+
+      workloadsPodPage.waitForPage();
+      workloadsPodPage.list().checkVisible();
+      workloadsPodPage.list().resourceTable().sortableTable().filter(clonePodName);
+      workloadsPodPage.list().resourceTable().sortableTable().rowWithName(clonePodName)
+        .checkExists();
+
+      // Simple test to assert we haven't broken Pods detail page
+      // https://github.com/rancher/dashboard/issues/10490
+      const clonedPodPage = new WorkLoadsPodDetailsPagePo(clonePodName);
+
+      clonedPodPage.goTo();// Needs to be goTo to ensure http request is fired
+      clonedPodPage.waitForPage();
+
+      cy.wait('@clonedPod', { timeout: 20000 })
+        .then(({ response }) => {
+          expect(response?.statusCode).to.eq(200);
+
+          const clonedSpec = response?.body?.spec;
+
+          // In Dashboard adds empty affinity object by default
+          // Remove this to compare
+          if (!Object.keys(clonedSpec.affinity).length) {
+            delete clonedSpec.affinity;
+          }
+
+          expect(clonedSpec).to.deep.eq(origPodSpec);
+          expect(clonedSpec.containers[0].resources).to.deep.eq(createPodBlueprint.spec.containers[0].resources);
         });
-      });
     });
   });
+
+  after(() => {
+    podNamesList.forEach((r) => cy.deleteRancherResource('v1', `pods/${ nsName }`, r, false));
```

Comment:
to save time, pods can be created in a new namespace and then just that namespace can be deleted. that would have been a good suggestion to make yesterday, apologies

Reply:
no worries! it was a great suggestion nonetheless

Reply:
updated

Reply:
I also snuck in a custom command that sets the resource list filtering/grouping. It was being done in multiple tests so I thought it would be beneficial to have a simple command for it. lmk what you think